src/tests/Common/XUnitLogChecker/XUnitLogChecker.cs

```diff
@@ -154,19 +153,25 @@ static int Main(string[] args)
             }
             else
             {
-                Console.WriteLine("[XUnitLogChecker]: The provided dumps path"
-                                + $" '{dumpsPath}' was not able to be read or"
-                                + " found. Skipping stack traces search...");
+                WriteLineTimestamp("The provided dumps path"
+                                 + $" '{dumpsPath}' was not able to be read or"
+                                 + " found. Skipping stack traces search...");
             }
         }
 
         // Rename the temp log to the final log, so that Helix can use it without
         // knowing what transpired here.
         File.Move(tempLogPath, finalLogPath);
-        Console.WriteLine("[XUnitLogChecker]: Finished!");
+        WriteLineTimestamp("Finished!");
         return SUCCESS;
     }
 
+    static void WriteLineTimestamp(string message)
+    {
+        Console.Write($"[XUnitLogChecker]: {System.DateTime.Now:HH:mm:ss.ff}: ");
```

Comment:
Is any concurrent console output possible? I assume not but if it is you might want to make this a single write to the console not two.